ui/src/containers/NodePageMetricsTab.js

```diff
@@ -104,7 +104,7 @@ const NodePageMetricsTab = ({
   instanceIP: string,
   controlPlaneInterface: string,
   workloadPlaneInterface: string,
-  nodesIPsInfo: [],
+  nodesIPsInfo: Object,
```

Comment:
I think we already have a type for that, I would say we should use it.

Reply:
Yes. that's totally right!